binds/python/bind_misc.cpp

```diff
@@ -250,6 +250,21 @@ void bind_misc(py::module& m) {
             [](morphio::Property::Annotation* a) { return a->_points._perimeters; },
             "Returns the list of perimeters of annotated points");
 
+    py::class_<morphio::Property::Marker>(m,
+                                          "Marker",
+                                          "Container class for NeuroLucida extra Markers ")
+        .def_property_readonly(
+            "label",
+            [](morphio::Property::Marker* marker) { return marker->_label; },
```

Comment:
since these are bare pointers, are the lifetimes being respected?  What happens in a morphology object has it's refcount decremented, will it live long enough that the Properties still exist?

Reply:
Good point. I have changed it so that it no longer use pointers but `const ref` instead.